<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AdcDacPB">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>48</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC-DAC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hiddenPB">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>76</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Hidden</string>
    </property>
   </widget>
   <widget class="QPushButton" name="couplerPB">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>104</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Coupler</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="xGroup">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>28</y>
      <width>84</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>X Axis</string>
    </property>
    <widget class="QPushButton" name="xMotorPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>20</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Motor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xCtrlPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>48</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Control</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xCSRPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>76</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>CSR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xRRegsPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>104</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>readReg</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xWtRamPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>132</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>writeRAM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xWtEEPPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>160</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>wtEEPROM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xFeaturePB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>188</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Feature</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="yGroup">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>28</y>
      <width>84</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Y Axis</string>
    </property>
    <widget class="QPushButton" name="yMotorPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>20</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Motor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yCtrlPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>48</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Control</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yCSRPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>76</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>CSR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yRRegsPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>104</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>readReg</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yWtRamPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>132</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>writeRAM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yWtEEPPB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>160</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>wtEEPROM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yFeaturePB">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>188</y>
       <width>68</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Feature</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>318</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
